qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
@didrocks and what if there already is a QOwnNotes binary in the $PATH (like the native QOwnNotes binary)?